package.json

```diff
@@ -8,7 +8,8 @@
     "/scss"
   ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "set -e && set -x && sass --style compressed scss/indigo-web.scss css/indigo-web.min.css && sass scss/indigo-web.scss css/indigo-web.css"
```

Comment:
the syntax of the command is from `build.sh`
I ran it and it does generate the CSS,
just not sure if I should still in the `set -e` and `set -x`. Not sure what that does

Reply:
The `set` commands tell bash to fail if a command fails, but github actions effectively does this, so you can drop them.

Reply:
cool, removed👍🏽